federatedscope/core/configs/cfg_attack.py

```diff
@@ -13,6 +13,30 @@ def extend_attack_cfg(cfg):
     cfg.attack.target_label_ind = -1
     cfg.attack.attacker_id = -1
 
+    # for backdoor attack
+
+    cfg.attack.edge_path = ''
+    cfg.attack.trigger_path = ''
+    cfg.attack.setting = 'fix'
+    cfg.attack.freq = 10
+    cfg.attack.insert_round = 100000
+    cfg.attack.mean = [0.1307]
```

Comment:
Why does the mean and std default to 0.1307 and 0.3081? Maybe they should be set according to cfg.data.type？